### ISSUE TITLE
Continues refresh when embedding history-graph

### ISSUE THREAD

**Issue by Vxider** [custom-cards/button-card#334]
**Checklist**
- [x] I updated the card to the latest version available
- [x] I cleared the cache of my browser

**Describe the bug**

The history graph card will continues refresh when embedded into button-card.

**Version of the card**
Version: 3.3.5

**To Reproduce**
This is the configuration I used:
```yaml
aspect_ratio: 1/2
custom_fields:
  cpu_proximity: |
    [[[
      return `<ha-icon
        icon="mdi:chip"
        style="width: 12px; height: 12px; color: deepskyblue;">
        </ha-icon><span>CPU Proximity:\t<span style="color: var(--text-color-sensor);">${states['sensor.macbook_pro_cpu_proximity'].state}°C</span></span>`
    ]]]
  cpu_peci: |
    [[[
      return `<ha-icon
        icon="mdi:chip"
        style="width: 12px; height: 12px; color: deepskyblue;">
        </ha-icon><span>CPU PECI:\t<span style="color: var(--text-color-sensor);">${states['sensor.macbook_pro_cpu_peci'].state}°C</span></span>`
    ]]]
  fan: |
    [[[
      return `<ha-icon
        icon="mdi:fan"
        style="width: 12px; height: 12px; color: deepskyblue;">
        </ha-icon><span>FAN:\t<span style="color: var(--text-color-sensor);">${states['sensor.macbook_pro_fan_left'].state}/${states['sensor.macbook_pro_fan_right'].state}</span></span>`
    ]]]
  battery: |
    [[[
      return `<ha-icon
        icon="mdi:battery"
        style="width: 12px; height: 12px; color: deepskyblue;">
        </ha-icon><span>Battery:\t<span style="color: var(--text-color-sensor);">${states['sensor.macbook_pro_battery'].state}%</span></span>`
    ]]]
  power: |
    [[[
      return `<ha-icon
        icon="mdi:power-plug"
        style="width: 12px; height: 12px; color: deepskyblue;">
        </ha-icon><span>Power:\t<span style="color: var(--text-color-sensor);">${states['sensor.macbook_pro_power_wattage'].state}W</span></span>`
    ]]]
  status:
    card:
      entities:
        - entity: sensor.macbook_pro
      hours_to_show: 24
      type: history-graph
      refresh_interval: 10
      style: |
        ha-card {
          box-shadow: none;
        }
  graph:
    card:
      entities:
        - entity: sensor.macbook_pro_cpu_peci
        - entity: sensor.macbook_pro_cpu_proximity
        - entity: sensor.macbook_pro_fan_left
        - entity: sensor.macbook_pro_fan_right
      hours_to_show: 1
      style: |
        ha-card {
          box-shadow: none;
        }
      type: history-graph
entity: group.macbook_pro
icon: 'mdi:laptop-mac'
name: MacBook
styles:
  card:
    - padding: 2%
    - font-size: 10px
    - text-transform: capitalize
    - overflow: unset
  custom_fields:
    cpu_proximity:
      - align-self: middle
      - justify-self: left
    cpu_peci:
      - align-self: middle
      - justify-self: left
    fan:
      - align-self: middle
      - justify-self: left
    battery:
      - align-self: middle
      - justify-self: left
    power:
      - align-self: middle
      - justify-self: left
    graph:
      - overflow: unset
    status:
      - overflow: unset
  grid:
    - grid-template-areas: >-
        "n n n" "i cpu_proximity cpu_peci" "i fan fan" "i battery power" "status
        status status" "graph graph graph"
    - grid-template-columns: 1fr 1fr 1fr
    - grid-template-rows: min-content min-content min-content min-content min-content 1fr
  icon:
    - color: |
        [[[
           if (entity.state == 'on') return 'lime';
           else return 'grey';
        ]]]
    - width: 100%
    - margin-top: 5%
  img_cell:
    - justify-content: center
    - align-items: center
    - margin: none
  name:
    - font-weight: bold
    - font-size: 13px
    - align-self: left
    - justify-self: left
    - padding-top: 1px
type: 'custom:button-card'
```

**Screenshots**

![screenshot](https://user-images.githubusercontent.com/6134315/82978292-cac73900-a016-11ea-92d8-a3188fe4f711.gif)

**Expected behavior**

Use history-graph outside the button-card will not have this problem.

**Desktop (please complete the following information):**
 - Browser: Chrome
 - Version: 83.0.4103.61

**Smartphone (please complete the following information):**

**Additional context**

**Comment by RomRider** [comment]
This is a known issue, I'll have to work on it at some point. Not super easy to fix, but I'll fix it :)

**Comment by RomRider** [comment]
Should now be fixed in 3.3.6, please give it a go :)
